src/strategies/music-os-accumulator/extractor.mjs

```diff
@@ -23,14 +23,6 @@ const strategies = [
     accumulator: (map) => {
       return (line) => {
         const data = JSON.parse(line);
-
-        const IPFSIANAScheme = "ipfs://";
-        if (data.erc721.tokenURI.includes(IPFSIANAScheme)) {
-          data.tokenURI = data.tokenURI.replace(
```

Comment:
my suspicion is that this will cause problems. because we're checking if `data.erc721.tokenURI` includes an IPFS scheme but when we're rewriting `data.tokenURI`. This is why I ended up rolling back @Kunkka0822's changes on that part of the code too...

Reply:
we can technically still merge it and try to find the error later

Reply:
I didn't get you. Basically, instead of having `https://neume.infura-ipfs.io/ipfs/...` we will now have `ipfs://...`.

Reply:
ok good

Reply:
```
jq '. | length' music-os-accumulator-including-sound-protocol
13315

jq '. | length' music-os-accumulator-extraction-before-sound-protocol
14506
```

this is now causing problems

Reply:
this might have fixed it: https://github.com/neume-network/strategies/commit/9690ae9f19f5b5f8ace076d57136867354228a31